include/crypto/evp.h

```diff
@@ -627,3 +630,19 @@ void evp_encode_ctx_set_flags(EVP_ENCODE_CTX *ctx, unsigned int flags);
 const EVP_CIPHER *evp_get_cipherbyname_ex(OPENSSL_CTX *libctx, const char *name);
 const EVP_MD *evp_get_digestbyname_ex(OPENSSL_CTX *libctx, const char *name);
 
+#ifndef FIPS_MODE
+/*
+ * Internal helpers for stricer EVP_PKEY_CTX_{set,get}_params().
```

Comment:
stricter

Reply:
Ok, something went wrong in `git` automation during my last rebase, as some of the nits here, including this, had been addressed already in a fixup commit.

I will squash things where appropriate and rebase on latest master once more, and make sure all of this are addressed again!

Reply:
Fixed in upcoming PR (false alarm in this case, what I fixed was another instance of the same typo!)